iotedgedev/azurecli.py

```diff
@@ -465,9 +465,11 @@ def create_iothub(self, value, resource_group, sku):
                 self.output.prompt(
                     "Creating IoT Hub. Please wait as this could take a few minutes to complete...")
 
-                result = self.invoke_az_cli_outproc(["iot", "hub", "create", "--name", value, "--resource-group",
-                                                     resource_group, "--sku", sku, "--query", "[].{\"IoT Hub\":name}", "--out", "table"],
-                                                    f("Could not create the IoT Hub {value} in {resource_group} with sku {sku}."), stdout_io=io, stderr_io=error_io)
+                cmd = ["iot", "hub", "create", "--name", value, "--resource-group", resource_group, "--sku", sku, "--query", "[].{\"IoT Hub\":name}", "--out", "table"]
+                if sku == "F1":
+                    cmd = cmd + ["--partition-count", "2"]
```

Comment:
>"2" [](http://example.com/codeflow?start=54&length=3)

could you please add a comment why was the default value set to 2? Documenting some context here will help with maintaining the code int he future.

Reply:
I can't locate the documentation where it is specified unfortunately. However, to verify this you can check the Azure Portal: when trying to create an F1 IoT Hub, it does not allow you to change the partition count on the Management Tab
![image](https://user-images.githubusercontent.com/1822123/169096273-c813a6e3-ebbf-4894-9a69-a2d19dfe39be.png)

Reply:
I see, thanks Roel for clarifying, Could you please add a comment for context here that says that the value is chosen for parity with Portal experience.